### ISSUE TITLE
Residues out of density after refinement

### ISSUE THREAD

**Issue by helgepat** [keitaroyam/servalcat#9]
I noticed that some of my side chains are out of density after running servalcat when they where in density before (purple):
![grafik](https://user-images.githubusercontent.com/97705752/207636356-483015b5-da89-4ab2-b968-63351789d413.png)
 I cannot spot a reason why this would happen (clashing residues pushing them out e. g.), so I wasy wondering if there is something that can be done to prevent this. Maybe it has something to do with the density being weaker in this spot compared to the surrounding region?
When I perform a real-space refinement on these residues in Coot it puts them right back into the density.

**Comment by keitaroyam** [comment]
It is a bit hard to tell from one picture. Is it possible to share coordinates of this residue only?

**Comment by helgepat** [comment]
I have extracted parts of the refined and parental model that went into refinement. I also included a portion of the map around the region of interest:

[model-refined.zip](https://github.com/keitaroyam/servalcat/files/10231684/model-refined.zip)

I am currently looking for some more issues with my models and saw that some secondary structure was destroyed (some 1-turn helices and such in loop regions). I compared here to the original model PDB-7K00.  I  think this comes from my modelling in Coot, but neverthelss I then noticed the residue mentioned above which is also in a 1-turn helix.

I am also looking into waters at the moment, I am aware that some are out density in the models I included, both before and after refinement. I still have to fix that...

**Comment by keitaroyam** [comment]
Thank you for the files. This is strange indeed. In terms of torsion angles (Refmac uses chi* torsion angles), the model before refinement is preferred. Could you first try the very latest Refmac? CCP4 8.0.007 was just released, which included Refmac5.8.0403. Several problems are fixed. Not sure it is relevant here, it's worth a try.

Water is sometimes tricky. Peak position may change by different sharpening levels. I would also check water omit Fo-Fc map. Steric clashes may also kick water out of density.

**Comment by helgepat** [comment]
I installed the newest CCP4 8.0.007, I still have to do the refinements, but I noticed that something is wrong with the restraints in Coot now? When I refine a stretch of RNA or protein it completely distorts the chain and rips it out of density?

**Comment by keitaroyam** [comment]
OMG.. you are right. I now tried Coot 0.9.8.6 with your model and some others, and yes something is wrong with RNA models (if I remove RNA from your model, protein refinement looks fine). If the new Refmac won't help, you can revert this update (uncheck update 007 from the update manager). Otherwise you can just copy new Refmac somewhere and revert the update. I will report it to Paul.

**Comment by helgepat** [comment]
I have now reverted to the previous release of CCP4 on my modelling computer and Coot is back in action. Thank you for the suggestion that reverting is possible, I did not know this!

I kept the new release on the workstation and performed a refinement of the model mentioned above, and I am happy to say that the new Refmac seems to have fixed this issue. Here is a picture of the new refinement vs the older release:
Green=parental model; Blue=new Refmac; Red=old Refmac
![grafik](https://user-images.githubusercontent.com/97705752/207860769-ea12f966-1cf0-4fac-a95c-e866272c945d.png)
